cypress/support/fragments/settings/calendar/library-hours/library-hours.js

```diff
@@ -0,0 +1,90 @@
+import { including } from '@interactors/html';
+import {
+  Button,
+  IconButton,
+  TextField,
+  Calendar,
+  PaneHeader,
+  Modal,
+  KeyValue,
+} from '../../../../../../interactors';
+import settingsMenu from '../../../settingsMenu';
+import dateTools from '../../../../utils/dateTools';
+
+const currentDate = dateTools.getCurrentDay();
+const initialEventName = 'Initial event name';
+const editedEventName = 'Edited event name';
+const nameAttributeValue = 'periodName';
+const currentEventLabel = 'Current:';
+
+const iconsSet = {
+  calendar: 'calendar',
+  edit: 'edit',
+};
+const selectors = {
+  calendarFirstCell: '.rbc-day-bg:first-child',
+  servicePoint: '[class^=navListSectionControl] [href^="/settings/calendar/library-hours/"]',
+};
+const pageHeaders = {
+  createEvent: 'New: Regular Library Hours Validity Period',
+  modifyEvent: 'Modify: Regular Library Hours Validity Period',
+  deleteEvent: 'Delete regular library hours',
+};
+const buttonLabels = {
+  new: 'New',
+  saveAndClose: 'Save & close',
+  delete: 'Delete',
+};
+const fieldLabels = {
+  from: 'Valid From:*',
+  to: 'Valid To:*',
+};
+
+export default {
+  createCalendarEvent() {
+    cy
+      .visit(settingsMenu.calendarLibraryHoursPath)
+      .get(selectors.servicePoint).first().click()
```

Comment:
You can use here NavListItem interacrtor

Reply:
@IhorBohdan I need to find the first list item in the list. And should not rely on `label` of that item. And in this case, I can not use `NavListItem` interactor